turbo.json

```diff
@@ -48,18 +48,31 @@
     "lint": {
       "outputs": []
     },
-    "check-types": {
+    "lint:staged": {
       "outputs": []
     },
-    "test": {
+    "check-types": {
       "outputs": []
     },
-    "lint:staged": {
+    "generate": {
+      "outputs": [
+        "graphql/index.ts",
+        "graphql.schema.json",
+        "saleor/api.tsx",
+        "src/graphql/index.ts"
+      ]
+    },
+    "test": {
       "outputs": []
     },
     "dev": {
       "cache": false
     }
   },
-  "globalDependencies": [".env", ".env.local"]
+  "globalDependencies": [
+    "pnp-lock.yaml",
+    ".env",
+    ".env.local",
+    "packages/eslint-config-checkout/index.js"
```

Comment:
Shouldn't this be a dependency of `lint` command only? 🤔

Reply:
Oh right, yes.

Reply:
Oh no, not right :D This file is a global dependency of all packages in the repo. Without this line, you'd need to run `--force` to force turborepo to run linter again when any rules inside `packages/eslint-config-checkout/index.js` change.